Tasks/VsTest/distributedtest.ts

```diff
@@ -147,17 +147,20 @@ export class DistributedTest {
 
         var consolidatedCiData = {
                     agentFailure: false,
+                    agentPhaseSettings: tl.getVariable('System.ParallelExecutionType'),
```

Comment:
confirm this variable is the used across build and release with Nitin ?

Reply:
Okay. Will check